sass/_lagar-regler.scss

```diff
@@ -0,0 +1,40 @@
+.lagar-page {
+}
+h1 {
+  font-size: 3em;
+}
+h2 {
+  color: white;
+}
+
+.lagarcontainer {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  background-color: green;
+  border: 2px solid black;
+
+  /*stilar för allt main content*/
+}
+
+.rightcolumn {
+  background-color: $primary-color;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  padding: 1em;
+  text-align: left;
+}
+a {
+  color: white;
+}
+p {
+  margin: 3px;
+}
```

Comment:
these are also global styles. move them inside rightcolumn if that's where they belong

Reply:
You mean move the styles from the .lagarcontainer to the .rightcolumn in my scss-file?